benchmarks/nel/scripts/datasets/download_mewsli-9.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -e
+
+# Downloads and sets up Mewsli-9 dataset.
+cd assets
+SUBDIR=dense_representations_for_entity_retrieval
+svn export https://github.com/google-research/google-research/trunk/$SUBDIR
+# Fix outdated requirement.
+sed -i -e 's/absl_py>=0.8.1/absl-py/g' $SUBDIR/requirements.txt
+# Only download and process English data.
+LANGS="ar de en es fa ja sr ta tr"
+LANG_LIST=($(echo $LANGS | tr ' ' "\n"))
```

Comment:
Minor point but do you need this variable? Since it's only used for iteration in bash I think it would behave the same as `$LANGS`?

Reply:
I decided to continue with the Mewsli-9 dataset in https://github.com/rmitsch/spacy-projects/pull/1. The newer version there drops this variable.
From what I remember, iterating over a string such as `$LANGS` behaves differently though than iterating over a list like `$LANG_LIST`.